scripts/gen_handles.py

```diff
@@ -127,6 +129,15 @@ def main():
 
     parsed_elf = ZephyrElf(args.kernel, edt, args.start_symbol)
 
+    if args.output_graphviz:
+        # Try and output the dependency tree
+        try:
+            dot = parsed_elf.device_dependency_graph('Device dependency graph', args.kernel)
+            with open(args.output_graphviz, 'w') as f:
+                f.write(dot.source)
+        except ImportError:
+            pass
```

Comment:
not a fan of lazy imports, but users should probably get a message here, e.g. `Device dependency graph not available (missing graphviz)`.

Reply:
I don't think this warrants polluting build logs with an unconditional `print`.
Making it opt-in with `--verbose` or similar defeats the purpose, because if someone is enabling debug to see the error they would already know to install `graphviz`.